deployments/deploy-ssh.sh

```diff
@@ -600,6 +600,8 @@ spec:
       value: "${TORPEDO_SSH_PASSWORD}"
     - name: TORPEDO_SSH_KEY
       value: "${TORPEDO_SSH_KEY}"
+    - name: AZURE_REGION
```

Comment:
Should this be Azure Endpoint?

Reply:
How do you differentiate between this and bucket region?

Reply:
@kshithijiyer-px Please confirm if I can modify this environment variable from AZURE_REGION to AZURE_STORAGE_ACCOUNT?

Reply:
@sabrarhussaini it will again be confusing let's make it `AZURE_ENDPOiNT`?

Reply:
Sure @kshithijiyer-px